apps/web/components/ui/PurchaseCard/PurchaseCard.vue

```diff
@@ -44,7 +44,7 @@
       :unit-name="productGetters.getUnitName(product)"
     />
     <UiBadges class="mt-4" :product="product" :use-availability="true" />
-    <div class="mt-2">
+    <div class="mt-2 variation-properties">
```

Comment:
why cant we use a tailwind class?

Reply:
Because the content is dynamic and we need to find that anchor element inside it and only color that one.